CompilerSource/ipc/codegen/Background_generated.h

```diff
@@ -0,0 +1,222 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+
+
+#ifndef FLATBUFFERS_GENERATED_BACKGROUND_H_
+#define FLATBUFFERS_GENERATED_BACKGROUND_H_
+
+#include "flatbuffers/flatbuffers.h"

```

Comment:
Where is this header coming from? We don't ship it on Windows. Is it in the AUR? The Debian repos?

Reply:
I haven't figured that out yet, I guess that's supposed to be the only file you need for the runtime or w/e.

Reply:
I'll just copy the comment from the plugin https://github.com/enigma-dev/lgmplugin/blob/796232a0b961db2fdc979f5a64ccdd6f2acbe95c/org/enigma/backend/util/Image.java#L21

BGRA is what GameMaker still is too, even in Studio.

Reply:
It does look like that's what it is from:
https://google.github.io/flatbuffers/

> Use flatc (the FlatBuffer compiler) to generate a C++ header (or Java/C#/Go/Python.. classes) with helper classes to access and construct serialized data. This header (say mydata_generated.h) only depends on flatbuffers.h, which defines the core functionality.

So we can do that, just wait until we get over to writing the EGM reading in C++ first.

Reply:
It does look like that's what it's from:
https://google.github.io/flatbuffers/

> Use flatc (the FlatBuffer compiler) to generate a C++ header (or Java/C#/Go/Python.. classes) with helper classes to access and construct serialized data. This header (say mydata_generated.h) only depends on flatbuffers.h, which defines the core functionality.

So we can do that, just wait until we get over to writing the EGM reading in C++ first.

Reply:
That's great. I don't have a problem with checking it in in this PR, if you like. I also don't mind waiting (but why would you check in the codegen and not its dependencies?).